package.json

```diff
@@ -44,12 +44,12 @@
     "eslint-plugin-n": "^16.3.0",
     "eslint-plugin-promise": "^6.1.1",
     "eslint-plugin-react": "^7.33.2",
-    "husky": "^8.0.3",
+    "husky": "^9.0.11",
     "jsdom": "^22.1.0",
     "lint-staged": "^15.0.2",
     "prettier": "3.0.3",
     "typescript": "^5.2.2",
     "vite": "latest",
     "vitest": "^0.34.6"
   }
-}
+}
```

Comment:
let's always include a new line at the end of files because most editors are configured that way. the change above removes that new line, and this should not happen